components/epaxos/src/command/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod command;
```

Comment:
```suggestion
mod command;
pub use crate::command::*;
```
这样写，调用的时候就不用写 command::command 了。